riff-cnb-clusterbuildtemplate.yaml

```diff
@@ -19,43 +19,48 @@ spec:
     - name: RUN_IMAGE
       description: The run image buildpacks will use as the base for IMAGE.
       default: packs/run:v3alpha2
+    - name: BUILDER_IMAGE
+      description: The builder image provides the buildpacks that will build the IMAGE from source.
+      default: projectriff/builder:latest
+    - name: UTIL_IMAGE
+      description: The util image provides pack utilities to analyze and export built images.
+      default: packs/util
```

Comment:
should we use `:latest` or pin to a release, like `:v3alpha1`?

Reply:
I would think a tag

Reply:
turns out packs/util is deprecated